src/api/routes/meetings.ts

```diff
@@ -44,34 +44,34 @@ const join = procedure
     // TODO: implement a lock option to the find query, details: https://stackoverflow.com/a/48297781
     // if the user's group is present in OngoingMeetings Model, return the meeting link 
   await sequelizeInstance.transaction(async (t) => {
-    const ongoingMeeting = await OngoingMeetings.findOne({ where: { groupId: group.id }, lock: true, transaction: t });
+    const ongoingMeeting = await db.OngoingMeetings.findOne({ where: { groupId: group.id }, lock: true, transaction: t });
     if (ongoingMeeting) {
       return ongoingMeeting.meetingLink;
     }
-  });
-  
-  // if not in OngoingMeetings, check if there are empty slots to use. Find a TM user id that is not in use and
-  // generate a meeting link using this user id.
-  const meetings = await OngoingMeetings.findAll({ attributes: ["tmUserId"] });
-  if (meetings.length >= apiEnv.TM_USER_IDS.length) {
-    await emailRoleIgnoreError("SystemAlertSubscriber", "超过并发会议上限", 
-      `上限：${apiEnv.TM_USER_IDS.length}。发起会议的分组：${ctx.baseUrl}/groups/${input.groupId}`, ctx.baseUrl);
-    return null;
-  }
+    
+    // if not in OngoingMeetings, check if there are empty slots to use. Find a TM user id that is not in use and
+    // generate a meeting link using this user id.
+    const meetings = await OngoingMeetings.findAll({ attributes: ["tmUserId"], lock: true, transaction: t });
```

Comment:
use `db.OngoingMeetings` everywhere and remove `import OngoingMeetings` from top of file.